File: backend/calApp/calorie-service/src/main/java/com/caliq/calorie_service/models/dto/UpdateUserDetailsDto.java
package com.caliq.calorie_service.models.dto;
import com.caliq.calorie_service.models.types.GoalType;
import com.caliq.calorie_service.models.types.SexType;
import java.math.BigDecimal;
import java.time.LocalDate;
public record UpdateUserDetailsDto(BigDecimal curentWeight,
                                   LocalDate birthDay,
                                   SexType sex,
                                   BigDecimal activityType,
                                   GoalType goalType,
                                   BigDecimal wantedWeight,
                                   BigDecimal height) {}

File: androidApp/NytriSync/app/src/main/java/com/ga3t/nytrisync/data/model/UpdateUserDetailsDto.kt
package com.ga3t.nytrisync.data.model
import java.math.BigDecimal
data class UpdateUserDetailsDto(
    val curentWeight: BigDecimal,
    val birthDay: String,
    val sex: String,
    val activityType: BigDecimal,
    val goalType: String,
    val wantedWeight: BigDecimal,
    val height: BigDecimal
)